### DISCUSSION TITLE
[ FF7 ]  Slow loading with Use of 4k and 8k Textures in battle

### DISCUSSION THREAD

**Discussion by Neffina** [julianxhokaxhiu/FFNx#164] [Q&A]
I noticed when using higher resolution textures over 2K, for example 4k or 8K, that there is a 2-3 second delay when loading into the battle.

**Comment by julianxhokaxhiu** [comment]
Hi thanks for the report. The issue itself may depend on the power of your own GPU, as well as the speed of the disk where you have it installed, the speed of the PCI-Ex your GPU works upon, the speed of your RAM, how many banks you have etc.

FFNx doesn't cache textures more than the engine requires. The engine itself was never developed to handle such large textures, therefore I can't do more to actually improve the situation sorry.

Cheers!
